log/log.go

```diff
@@ -0,0 +1,221 @@
+package log
+
+import (
+	"github.com/getsentry/raven-go"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var (
+	logger = zap.NewNop().Sugar()
+)
+
+// Level is the verbose representation of log level
+type Level string
+
+// Enums for Level
+const (
+	NopLevel   Level = "nop"
+	DebugLevel Level = "debug"
+	InfoLevel  Level = "info"
+	WarnLevel  Level = "warn"
+	ErrorLevel Level = "error"
+	PanicLevel Level = "panic"
+	FatalLevel Level = "fatal"
+
+	// This will have the same effect as nop but slower
+	ZapNopLevel zapcore.Level = zapcore.FatalLevel + 1
+)
+
+// InitLogger provides a quick way to start or replace a logger
+// Pass in debug as log level enables development mode (which makes DPanicLevel logs panic).
+func InitLogger(logLevel Level) {
+	if logLevel == NopLevel {
+		logger = zap.NewNop().Sugar()
+		return
+	}
+
+	var config = zap.NewProductionConfig()
+	lvl := StringToAtomicLevel(logLevel)
+	config.Encoding = "console"
+	config.EncoderConfig.EncodeCaller = ShortCallerEncoder
+	config.EncoderConfig.EncodeTime = TimeEncoder
+	config.EncoderConfig.EncodeLevel = CapitalLevelEncoder
+	config.Level = zap.NewAtomicLevelAt(lvl)
+
+	if lvl == zap.DebugLevel {
+		config.Development = true
+	}
+	// TODO: error handling
+	l, _ := config.Build(zap.AddCallerSkip(2))
+	logger = l.Sugar()
+}
```

Comment:
In order to reduce duplicated code, I'd prefer we call `InitLoggerWithConfig` from `InitLogger`. Basically in `InitLogger` we just create the config and then call `InitLoggerWithConfig`.